api/src/middleware/cloudflare-filter/cloudflare-filter.middleware.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable, NestMiddleware, HttpException, HttpStatus } from '@nestjs/common';
+import { Request, Response } from 'express';
+import { CloudflareIpService } from '../../services/cloudflareip/cloudflare-ip.service';
+import { inRange } from 'range_check';
+
+@Injectable()
+export class CloudflareFilterMiddleware implements NestMiddleware {
+  constructor(private cloudflareIpService: CloudflareIpService) {}
+
+  use(req: Request, res: Response, next: (any?) => void) {
+    if (process.env.ENVIRONMENT !== 'dev') {
```

Comment:
`NODE_ENV=development` vs. `NODE_ENV=production` is the standard so I'd keep your code the same, but simply rename `ENV` to `NODE_ENV`